### PR TITLE
Buff up Admewn and BDH93

### PR DESCRIPTION
**Pull request by TheRun98**

### CODE REVIEW

File: mods/sgssb/moves.js [hunk 1]
```diff
@@ -418,12 +418,24 @@ exports.BattleMovedex = {
 		isNonstandard: true,
 		name: "Mewtation",
 		pp: 10,
+		secondary: {
+			chance: 100,
```
Comment (ghost) [inline mods/sgssb/moves.js]: If you do `chance: 100,` Might as well take the thing out of secondary and put in `status: 'tox ',` and put `secondary: false,`

Comment (ghost) [reply mods/sgssb/moves.js]: This is just me nitpicking but if a poke with shield dust came it. The opponent wouldn't get toxic'd

**Comment by TheRun98** [comment]
Shield Dust wouldn't matter because it is not in the metagame at this point.

File: mods/sgssb/moves.js [hunk 2]
```diff
@@ -418,12 +418,22 @@ exports.BattleMovedex = {
...
 		onPrepareHit: function (target, source) {
 			this.attrLastMove('[still]');
-			this.add('-anim', source, "Transform", source);
+			this.add('-anim', source, "Agility", source);
+			this.add('-anim', source, "Psychic", target);
+			this.add('-anim', source, "Night Shade", target);
 		},
-		desc: "No additional effects",
+		desc: "Boosts accuracy by 1 and poisons target.",
+		shortDesc: "Boosts accuracy by 1 and poisons target.",
```
Comment (ghost) [inline mods/sgssb/moves.js]: Forgot Acc -> Evasion Change

**Review by ghost — APPROVED** [review]
Everything looks good to merge @HoeenCoder. Just add badly poisons to Mewtation's Desc

**Review by DeathlyPlays — APPROVED** [review]

**Pull request merged**